front/app/services/adminPublications.ts

```diff
@@ -41,6 +41,7 @@ export interface IAdminPublicationData {
       data: IRelationship[];
     };
     parent: {
+      // The id here is the parent's publication id, *not* the folder id.
```

Comment:
Minor: The comment here is about the id but the next attribute is data. Just making sure that that is intentional.

Reply:
Yes, it's the id in the IRelationship. Clarifying this in the other PR.